### PR TITLE
docs: new prompt, changed readme, upload image, rename image

### PR DESCRIPTION
**Pull request by onixldlc**
# Description
- added more prompt in prompt_demo.txt
- removed other_output section from README.md and rename `output` section to `examples`
- change image shown in the README.md file
- upload 2 more generated image
- renamed the `minimalist_house_with_family_mountainous_forested_wild_concept art illustration.png` to `minimalist_house_with_family_mountainous_forested_wild_concept_art_illustration.png`
- fix code example in README.md (used StableDiffusion class instead of the deprecated Text2img class )

# Environment

- ``OS`` = `Linux`

**linked issue** : #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,55 +102,13 @@ generator = Text2Image(
...
-| prompt: harry potter playing guitar     | ![harry_potter_playing_guitar](.github/result_output/harry_potter_playing_guitar.png)     |
-| prompt: person hacking with computer    | ![person_hacking_with_computer](.github/result_output/person_hacking_with_computer.png) |
-| prompt: God wearing mask | ![god_wearing_mask](.github/result_output/god_wearing_mask.png) |
-| prompt: Harry potter random | ![harry_potter_random](.github/result_output/harry_potter_random.png) |
-| prompt: Marilyn monroe with random  art style | ![marilyn_monroe_with_random_art_style](.github/result_output/marilyn_monroe_with_random_art_style.png) |
+| minimalist house with family, mountainous forested wild, concept art illustration | ![minimalistic_house](.github/result_output/minimalist_house_with_family_mountainous_forested_wild_concept_art_illustration.png) |
+| natural cave wall, dynamic light, mist low over ground, illustration by josan gonzales and moebius, studio muti, malika favre, rhads, makoto, clean thick line, comics style | ![natural_cave](.github/result_output/natural_cave.png) |
```
Comment (slowy07) [inline README.md:110]: can you adding ignore word ``muti`` on unitestting ,because this word get caught by codespell

**Review by slowy07 — COMMENTED** [review]
request change on codespell

File: README.md (hunk 1 above)
Comment (onixldlc) [reply README.md:110]: ok

**Review by slowy07 — APPROVED** [review]
LGTM, nice one @onixldlc

**Pull request merged**